enterprise-suite/scripts/lbc.py

```diff
@@ -184,11 +213,81 @@ def preinstall_check(creds, minikube=False, minishift=False):
         fail('Your credentials do not appear to be correct' +
                  ' - unable to make authenticated request to lightbend docker registry')
 
-def install_helm_chart(creds_file):
+# Returns one of 'deployed', 'failed', 'pending', 'deleting', 'notfound' or 'unknown'
+def install_status(release_name):
+    stdout, returncode = run('helm status ' + release_name,
+                            DEFAULT_TIMEOUT, show_stderr=False)
+    if returncode != 0:
+        return 'notfound' 
+    
+    if 'STATUS: DEPLOYED' in stdout or (stdout == ''):
+        return 'deployed'
+    if 'STATUS: FAILED' in stdout:
+        return 'failed'
+    if 'STATUS: PENDING_INSTALL' in stdout:
+        return 'pending'
+    if 'STATUS: PENDING_UPGRADE' in stdout:
+        return 'pending'
+    if 'STATUS: DELETING' in stdout:
+        return 'deleting'
+    return 'unknown'
+
+# Returns true if console PVCs are already present in the namespace.
+# Exits with an error if some PVCs are present, but not all.
+def are_pvcs_created(namespace):
+    stdout, returncode = run('kubectl get pvc --namespace={} --no-headers'.format(namespace))
+    if returncode == 0:
+        all_found = True
+        found_pvcs = []
+        for pvc in CONSOLE_PVCS:
+            if pvc in stdout:
+                found_pvcs.append(pvc)
+            else:
+                all_found = False
+
+        if not all_found and len(found_pvcs) > 0:
+            fail('Found some PVCs from previous console install, but not all: {}.\nTo avoid data loss, please clean them up manually'
+                 .format(str(found_pvcs)))
+
+        return all_found
+    return False
+
+# Returns true if required cluster roles are already created.
+# Exits with an error if some cluster roles are present, but not all.
+def are_cluster_roles_created():
+    stdout, returncode = run('kubectl get clusterroles --no-headers')
+    if returncode == 0:
+        all_found = True
+        found_crs = []
+        for cr in CONSOLE_CLUSTER_ROLES:
+            if cr in stdout:
+                found_crs.append(cr)
+            else:
+                all_found = False
+
+        if not all_found and len(found_crs) > 0:
+            fail('Found some cluster roles from previous console install, but not all: {}. Please clean up manually.'
+                 .format(str(found_crs)))
+
+        return all_found
+    return False
+
+def install(creds_file):
     creds_arg = '--values ' + creds_file
+    version_arg = ('--version ' + args.version) if args.version != None else '--devel'
```

Comment:
The `install-es-sh` script would allow `--version=blah` as well as `--version blah`.  (Only cuz helm does.)  Does this do the same?  (I'm no python expert.)  I don't think so, and frankly I think that's okay.  I only mention it in case someone does think it matters.

Reply:
Just tried it out - yes, it supports both.